pkg/performanceprofile/cmd/render/cmd.go

```diff
@@ -55,7 +56,7 @@ func NewRenderCommand() *cobra.Command {
 	}
 
 	renderOpts.AddFlags(cmd.Flags())
-
+	addKlogFlags(cmd)
```

Comment:
(semi-)nit: klog flags, or in general flags pertaining to utility modules, are usually added first, then the application-specific flags.

Reply:
Done. Thanks